### PR TITLE
Find optical tip

### PR DESCRIPTION
**Pull request by Rmano**
Ok, this is a try to solve #302, but I think I will not merge it. The problem is that now the point will not overshoot, but you can see a bit of the "stem": 

image

which I find equally ugly... avoiding this is difficult, and the only way would be to use proper arrows for transistors; but then they won't scale (tikz arrows scale with linewidth, not with scale).

### CODE REVIEW

**Comment by Rmano** [comment]
Ok, I had an idea. To avoid the ugly line overrunning, I can retract the connection line a bit. I tried it on BJTs, they are the most difficult: 

image

The arrow now lies in the middle of the connection, which I do not know if it's nice: 
image

The old situation is this: 

image

...which is also ugly.

This is with transparent elements that show what's happening: 

image

**Comment by Rmano** [comment]
...on the other hand, the solution of simply shifting the arrow as in the first image of the PR is not so bad. The wires show better connection and the wires sticking out the point of the arrow are not so disturbing in normal magnification...

**Comment by BurakKelleci** [comment]
I think retracting the line a bit is fine. I checked couple of books for MOS and NPN drawings. Their npn drawing turns out to be problematic.
image
This is NMOS drawing

image
This is NPN drawing. I think circuitikz drawing is better than this.

**Comment by Rmano** [comment]
Yes, I also like better the `circuitikz` style. The question for me now is... should I retract the line or not? On one hand, sometimes retracting seems better. On the other hand, the line continuity symbolize the electric connectivity, and when I draw manually the circuit, I normally draw the lines and then add the arrows. 

Also, in type-p arrows (when they touch the gate, which can be thicker) I do not know if the arrow should just barely touch it(*) or continue to be like now, that touch the middle line of the gate. 

(*) the problem with barely touching is that at normal magnification, and with the anti-aliasing thing, it can seem disconnected. 

Basically, I quite like the status quo now... I have to look for the retracting or not for MOSes.

**Comment by Rmano** [comment]
I think #305 is simpler and a better approach. Closing.
